lib/python/temporal/core.py

```diff
@@ -650,21 +651,30 @@ def init(raise_fatal_error=False, skip_db_version_check=False):
         if dbif.fetchone()[0]:
             db_exists = True
 
-    backup_howto = "The format of your actual temporal database is not " \
-                   "supported any more.\nSolution: You need to export it by " \
-                   "restoring the GRASS GIS version used for creating this DB"\
-                   ". From there, create a backup of your temporal database "\
-                   "to avoid the loss of your temporal data.\nNotes: Use " \
-                   "t.rast.export and t.vect.export to make a backup of your" \
-                   " existing space time datasets.To safe the timestamps of" \
-                   " your existing maps and space time datasets, use " \
-                   "t.rast.list, t.vect.list and t.rast3d.list. "\
-                   "You can register the existing time stamped maps easily if"\
-                   " you export columns=id,start_time,end_time into text "\
-                   "files and use t.register to register them again in new" \
-                   " created space time datasets (t.create). After the backup"\
-                   " remove the existing temporal database, a new one will be"\
-                   " created automatically.\n"
+    backup_howto = _("The format of your actual temporal database is not " \
+                     "supported any more.\n" \
+                     "From there, create a backup of your temporal database "\
+                     "to avoid the loss of your temporal data.\n")
+    if tgis_db_version > 2:
+        backup_howto += _("Recommended solution: Run t.upgrade command installed from "\
+                          "GRASS Addons.\n")
+        backup_howto += _("Alternative solution: ")
+    else:
+        backup_howto += _("Solution: ")
+
+    backup_howto += _("You need to export it by " \
+                      "restoring the GRASS GIS version used for creating this DB."\
```

Comment:
I don't understand this part: do you mean export the temporal database "to" restore the grass version? If yes, then I'd replace the sentence with: "Export your temporal databases to restore the GRASS GIS version"

Reply:
This part has been written by @huhabla